packages/database/src/realtime/Constants.ts

```diff
@@ -25,7 +25,8 @@ export const REFERER_PARAM = 'r';
 
 export const FORGE_REF = 'f';
 
-export const FORGE_DOMAIN = 'firebaseio.com';
+// Matches console.firebase.google.com and *.corp.google.com.
+export const FORGE_DOMAIN_RE = /(console\.firebase|.*\.corp)\.google\.com/;
 
```

Comment:
Can we make the regexp stricter to match firebase console under corp domain only? I'm worried it may brake some internal apps.

Reply:
Updated to use firebase-console-\w*.corp as per Laura's suggestion.

Reply:
Now it doesn't match the prod firebase corp URL: "firebase.corp.google.com" :-).

Reply:
It's a regular expression, so of course it is going to be buggy. Fixed (and manually tested)